python/test/ngraph/test_basic.py

```diff
@@ -62,11 +62,14 @@ def test_serialization():
     model = (parameter_a + parameter_b) * parameter_c
     runtime = ng.runtime(backend_name=backend_name)
     computation = runtime.computation(model, parameter_a, parameter_b, parameter_c)
-    serialized = computation.serialize(2)
-    serial_json = json.loads(serialized)
-
-    assert serial_json[0]['name'] != ''
-    assert 10 == len(serial_json[0]['ops'])
+    try:
+        serialized = computation.serialize(2)
+        serial_json = json.loads(serialized)
+
+        assert serial_json[0]['name'] != ''
+        assert 10 == len(serial_json[0]['ops'])
+    except Exception:
```

Comment:
Let's narrow this exception down to the type we would expect when `NGRAPH_JSON_ENABLE=OFF`

Reply:
Let's leave that as a follow-up. I added a JIRA.